uiowa/uiowa
# Cleanup outdated code

# Updates

* Remove `media_document` module
* Remove `media_image` module
* Remove `media_video` module
* Remove `sitenow_v2_default_content` module
* Re-factor how page title is displayed on page title hero block to not use Twig Tweak method
* Updated collegiate split to remove `media_core` and `twig_tweak` modules and to add a new permission that was missed in a previous update.
* Fixed PHPCS warning from #2497 

# How to test

* `blt ds` to test no issues with syncing default site.
* `blt ds --site=cogscilang.grad.uiowa.edu` to test no issues with syncing a collegiate site.
* `blt ds --site=uiowa.edu`
* Visit https://uiowa.local.drupal.uiowa.edu/research and see that it loads without issues - this page uses the page title hero.
* The `media_core` module was not providing any functionality, so removing it should not have any effect.
* The other custom modules were not in use.
